packages/devtools_app/lib/src/ui/flutter/flutter_icon_renderer.dart

```diff
@@ -0,0 +1,140 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Library to render [DevToolsIcon]s as [Widget]s.
+library icon_renderer;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import '../icons.dart';
+import '../material_icons.dart';
+import '../theme.dart';
+
+final Expando<Widget> widgetExpando = Expando('IconRenderer');
```

Comment:
This matches the html_icon_renderer file.
I haven't tested all the functionality.
The best way to test this would be with flutter golden image tests.

Reply:
great!

Reply:
A description of what's needed: https://github.com/flutter/devtools/issues/1195

If we don't want to load the icon fonts and only test the logic for how we draw the icons, then this will be a four-line change to implement, and we'll do updates the same way as already.  Feel free to try it out.

I don't want to land any golden image tests for any other part of the app yet because most of it will be changing so fast that goldens will only get in the way.